website/docs/docs/dbt-versions/2024-release-notes.md

```diff
@@ -9,6 +9,10 @@ pagination_prev: null
 
 dbt Cloud release notes for 2024.
 
+## May 2024
+
+- **Behavior change:** Introduced the [`require_resource_names_without_spaces`](/docs/reference/global-configs/legacy-behaviors#require_resource_names_without_spaces) flag, opt-in and disabled by default. If set to `True`, dbt will raise an exception if it finds a resource name containing a space in your project or an installed package. This will become the default in a future version of dbt.
```

Comment:
it looks like vercel isn't liking the achor urls @jtcohen6 
![Screenshot 2024-05-01 at 14 43 10](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/fd4ea930-d2ab-493f-8da9-6e5f5687ef5c)

Reply:
I think the changes Mirna suggests will fix this.